Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_On_Prem_Remediation.yml

```diff
@@ -215,7 +393,19 @@ inputs:
   required: true
   description: Port number of the service.
   playbookInputQuery:
+- key: DeviceGroup
+  value: {}
+  required: false
+  description: Device Group of the firewall rule to lookup
+  playbookInputQuery:
+- key: SecondaryDeviceGroup
+  value: {}
+  required: false
+  description: In that case that the rule, address and service are created in the "Shared" location, we need to know what device-groups we can push to because it isn't possible to push to the "Shared" location.
```

Comment:
```suggestion
  description: If the rule, address and service are created in the "Shared" location, we need to know what device groups we can push to because it isn't possible to push to the "Shared" location.
```